native/src/components/Header.tsx

```diff
@@ -82,14 +86,45 @@ const Header = ({
   const [previousRoute] = useState(navigation.getState().routes[navigation.getState().routes.length - 2])
   const [canGoBack] = useState(navigation.canGoBack())
 
+  const getShareTitle = (): string => {
+    const pageTitle = (route.params as { title?: string } | undefined)?.title
+    if (!previousRoute) {
+      // Home/Dashboard: Show city name
+      return cityName ?? ''
+    }
+
+    if (route.name === POIS_ROUTE && previousRoute.name === POIS_ROUTE) {
+      const poisRouteParams = route.params as RoutesParamsType[PoisRouteType]
+      if (poisRouteParams.slug || poisRouteParams.multipoi) {
+        return t('locations')
```

Comment:
you are right. this check is only needed for multipois to use sth like 'Gefunden: Mehrere Orte - <Location> <Link>'
The normal title is already set in the first line of the fkt

Reply:
yes we should update this also on web but thats couple of additional work which i would do in a separate task, since its not that easy to get the poi title. You can parse the url but thats not a nice solution

Reply:
Actually I think I already (accidentally) fixed that in #2671 by fixing the page title for pois.